userspace/libsinsp/parsers.cpp

```diff
@@ -2249,8 +2270,11 @@ inline void sinsp_parser::add_socket(sinsp_evt *evt, int64_t fd, uint32_t domain
 	}
 	else
 	{
-		if(     domain != 10 && // IPv6
-		        domain != 17)   // AF_PACKET, used for packet capture
+		if(domain != 10 && // IPv6
+#ifdef _WIN32
+			domain != AF_INET6 && // IPv6 on Windows
+#endif
+			domain != 17)   // AF_PACKET, used for packet capture
```

Comment:
Can this be simply

```suggestion
		if(domain != AF_INET6 && // IPv6
			domain != 17)   // AF_PACKET, used for packet capture
```

?

Reply:
probably. I didn't want to touch the linux code because those comparisons against numbers instead of the properly defined constants looked like someone had done it that way for a reason. If you promise me it won't break something, I'm happy to make the change. :-)